instrumentation/instagocb/go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/instana/go-sensor/instrumentation/instagocb
+
+go 1.13
+
+require (
+	github.com/couchbase/gocb/v2 v2.6.5
+	github.com/couchbase/gocbcore/v10 v10.2.9
+	github.com/instana/go-sensor v1.58.0
+	github.com/opentracing/opentracing-go v1.2.0
+	github.com/stretchr/testify v1.8.2
+)
+
+replace github.com/instana/go-sensor => ../../
```

Comment:
I'm keeping this replace statement here to prevent test case failures in the current pull request. (As we've implemented some changes in the core module for couchbase span support)

I am planning to remove it in a subsequent pull request before releasing instagocb.